src/ImageSharp/Formats/Bmp/BMP Native Structures/CIEXYZ.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.Runtime.InteropServices;
+
+namespace SixLabors.ImageSharp.Formats.Bmp
+{
+    /// <summary>
+    /// The <c>CIEXYZ</c> structure contains the X, Y, and Z coordinates of a specific color in a
+    /// specified color space (CIE XYZ 1931).
+    /// <para>
+    /// X, Y and Z are extrapolations of RGB created mathematically to avoid negative numbers
+    /// (In 1931 there weren�t any computers) and are called Tristimulus values.
+    /// Y means luminance, Z is somewhat equal to blue, and X is a mix of cone response curves
+    /// chosen to be orthogonal to luminance and non-negative.
+    /// <seealso href="http://wolfcrow.com/blog/what-is-the-difference-between-cie-lab-cie-rgb-cie-xyy-and-cie-xyz">See this Wolfcrow link for more information.</seealso>
+    /// </para>
+    /// <para>Supported since Windows 2000 and Windows 98</para>
+    /// <para>Implemented on Microsoft Windows BMP v4 format.</para>
+    /// </summary>
+    /// <remarks>
+    /// Make shore that <c>sizeof(CIEXYZ)</c> returns the size of 12 bytes and is byte aligned.
+    /// All structure fields are stored little-endian on the file.
+    /// <para>
+    /// <seealso href="https://msdn.microsoft.com/en-us/library/dd371828(v=vs.85).aspx">See this MSDN link for more information.</seealso>
+    /// </para>
+    /// </remarks>
+    [StructLayout(LayoutKind.Sequential, Pack = 1, Size = 12)]
+    internal struct CIEXYZ
```

Comment:
How does this relate to the [CieXyz](https://github.com/SixLabors/ImageSharp/blob/2c8d68ae3d65eacf9123aeedfc63335f6e936ff9/src/ImageSharp/ColorSpaces/CieXyz.cs) struct already present in the library. Can we reuse that?

Reply:
I will check if there is any difference. If not I will use that class.

Reply:
They are implementation diferent but functionaly the same.

You implenment it as floats for X, Y, and Z, Microsoft implemented them as fixed points in the form 2.30 encoded in DWORD. See:

https://msdn.microsoft.com/en-us/library/windows/desktop/dd371828(v=vs.85).aspx

So I will read/write them in Microsoft format and store them internally as your CieXyz class. That should allow the comm code of the framework to "fix" the colors for the output device (screen, printer, etc.).

Reply:
@JimBobSquarePants @caetanator 
I think we should keep these structures as data transfer objects as similar to the standard as possible. I see no advantage from utilizing the ImageSharp colorspace stuff here.